jetty-servlets/src/main/java/org/eclipse/jetty/servlets/CrossOriginFilter.java

```diff
@@ -63,16 +62,33 @@
  * <p>
  * Allowed origins can therefore be more complex expressions such as
  * https?://*.domain.[a-z]{3} that matches http or https, multiple subdomains
- * and any 3 letter top-level domain (.com, .net, .org, etc.).</dd>
- * 
+ * and any 3 letter top-level domain (.com, .net, .org, etc.).
+ * <p>
+ * Won't be used if <b>allowedOriginsRegex</b> is declared.
```

Comment:
If `allowedOrigins` is deprecated, then we should produce a warning if it is used.
However, I think I prefer that we keep it undeprecated and:
 - warn if it contains '*' character, but still process it as a regex.
 - error if it contains a '*' character and allowedOriginsRegex is not null